test/test_api.py

```diff
@@ -4,8 +4,14 @@
 
 if sys.version_info[0] < 3:
     STRING_TYPE = basestring
+    PATH_TYPE = basestring
+elif sys.version_info[1] < 4:
```

Comment:
Careful with this check... python4 may come

Reply:
wait, that is the second element... still though python 4.0 would enter this code path, which is (likely) incorrect

Reply:
perhaps try/except `import pathlib` instead?